Dockerfile

```diff
@@ -54,7 +54,7 @@ RUN \
 
 # Install GHC.
 
-ARG GHC_VERSION=9.0.2
+ARG GHC_VERSION=9.2.4
```

Comment:
Are we not changing the default yet? I changed this because I assumed we were.

Reply:
I hadn't changed the default yet because I wanted the default no-arg Docker build to work: https://github.com/acilearning/docker-haskell/pull/5#discussion_r942507479